src/tokenserver/handlers.rs

```diff
@@ -35,7 +34,7 @@ pub struct TokenserverResult {
 pub async fn get_tokenserver_result(
     req: TokenserverRequest,
     db: Box<dyn Db>,
-    mut metrics: Metrics,
+    TokenserverMetrics(mut metrics): TokenserverMetrics,
```

Comment:
Recently learned you could destructure in function arguments 🦀

Reply:
TIL. neat!